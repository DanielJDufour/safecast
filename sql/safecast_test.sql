CREATE EXTENSION safecast;

SELECT to_integer('1sasdf');
SELECT to_integer('123123');

SELECT to_float('0.123asfd');
SELECT to_float('0.123');

SELECT to_bigint('a12345678x9123456789z');
SELECT to_bigint('123456789123456789');

SELECT to_double_precision('0.123asfd');
SELECT to_double_precision('0.123');
SELECT to_double_precision('12.123');
