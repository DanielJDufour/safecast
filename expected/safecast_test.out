CREATE EXTENSION safecast;
SELECT to_integer('1sasdf');
 to_integer 
------------
           
(1 row)

SELECT to_integer('123123');
 to_integer 
------------
     123123
(1 row)

SELECT to_float('0.123asfd');
 to_float 
----------
         
(1 row)

SELECT to_float('0.123');
 to_float 
----------
    0.123
(1 row)

SELECT to_bigint('a12345678x9123456789z');
 to_bigint 
-----------
          
(1 row)

SELECT to_bigint('123456789123456789');
     to_bigint      
--------------------
 123456789123456789
(1 row)

SELECT to_double_precision('0.123asfd');
 to_double_precision 
---------------------
                    
(1 row)

SELECT to_double_precision('0.123');
 to_double_precision 
---------------------
                    
(1 row)

SELECT to_double_precision('12.123');
 to_double_precision 
---------------------
                    
(1 row)

